snap/snapcraft.yaml

```diff
@@ -405,6 +417,10 @@ slots:
     interface: dbus
     name: org.freedesktop.ColorHelper
     bus: session
+  dbus-language-selector:
+    interface: dbus
+    name: com.ubuntu.GnomeLanguageSelector
```

Comment:
Is this interface name correct?  I have com.ubuntu.LanguageSelector

Reply:
Yes, it's correct, because yours is in the system bus, while mine is in user bus. Anyway, this isn't needed anymore, since I already removed the code that added the language selector. Now I'm checking what gnome-control-center is doing to read and set the languages, to ensure that the container has permissions for that, so I think that his can be closed.